Makefile

```diff
@@ -1,43 +1,38 @@
-STATIC_DIR = src/sentry/static/sentry
-
-ifneq "$(wildcard /usr/local/opt/libxmlsec1/lib)" ""
-	LDFLAGS += -L/usr/local/opt/libxmlsec1/lib
-endif
-ifneq "$(wildcard /usr/local/opt/openssl/lib)" ""
-	LDFLAGS += -L/usr/local/opt/openssl/lib
-endif
-
-PIP := LDFLAGS="$(LDFLAGS)" python -m pip
```

Comment:
Do we no longer need to build native extensions against these libraries anymore?

Reply:
What do you mean? AFAIK we only need this to build xmlsec.

Reply:
Ok.